README.md

```diff
@@ -32,43 +32,13 @@ Note that the following covers only local configuration, not deployment. Neverth
  `src` for how we deploy to [Elastic Beanstalk](https://aws.amazon.com/elasticbeanstalk/) using 
```

Comment:
Yes, I agree. Let's clean this up in another card? I'd prefer to keep these tasks small... :)

Reply:
Yep that sounds good. I think it would be most efficient if you just create a card  in Backlog every time you come across something that we need to remember to do, and then yes we can keep tasks small.